src/lib/sorting.ts

```diff
@@ -1,7 +1,7 @@
 import { RNPlugin } from '@remnote/plugin-sdk';
 
 export const DEFAULT_RANDOMNESS = 0;
-export const DEFAULT_RATIO = 0.25;
+export const DEFAULT_RATIO = 0.1;
```

Comment:
I think I'll keep the ratio at 0.25. Users can customize this if they don't like it :+1: